### PR TITLE
Portfolio: mobile menu

### PR DESCRIPTION
**Pull request by devendra-alt**
**[Portfolio: mobile menu](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md)**

- Added mobile menu page
- Added javascript for trigger events on the open and close menu button
- Added HTML, CSS, javascript linters

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+const closeMenu = () => mobileMenu.classList.remove('open-menu');
+menuItems.forEach((el) => {
+  el.addEventListener('click', () => closeMenu());
+});
+menuBtn.addEventListener('click', () => {
+  openMenu();
+});
+closeBtn.addEventListener('click', () => {
+  closeMenu();
+});
```
Comment (shella12) [inline index.js:15]: - [x] Please make sure when the user expands to desktop size the hamburger menu button is not visible. 

Screenshot 2023-05-22 at 10 34 14 PM

File: index.html [hunk 2]
```diff
@@ -25,21 +37,7 @@
           <li><a href="" class="nav-link">About</a></li>
           <li><a href="" class="nav-link">Contact</a></li>
```
Comment (shella12) [inline index.html]: - [x] Please make sure when the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed. By prepending` href` with` # `, you can target an HTML element with a specific id attribute. For example,` <a href="#contact">` will navigate to the `<section id="contact">` within the same HTML document. This type of `href` is often used to navigate back to the top of the page.

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @devendra-alt and @sadaf-Daneshgar, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- Menu closes on clicking (X) ✅ 
- Menu opens on clicking hamberger ✅   
- Design is implemented correctly ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (devendra-alt) [reply index.html]: hello @shella12, thank you for your suggestions, I have completed all the required changes.

**Review by Davidosky007 — APPROVED** [review]
Hi @devendra-alt 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* Proper use of DOM manipulation 👍 
* JavaScript syntax used 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**